test/examples/fail_line_length.erl

```diff
@@ -9,6 +9,7 @@
          function_6/2
         ]).
 
+% Single line comment over 80 characters!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
I actually discovered a bug in the default line length rule while attempting to add these test cases. On `master` branch (specifically commit `f8b022d4`) I found that lines starting with tabs and exceeding 80 "characters" in length were not being raised as style problems. I quote the word "characters" because the length is dependent on how many characters a single tab stands for. For example, I added a bunch of tabs to the front of line 13 in `test/examples/fail_line_length.erl` which should have pushed the "length" to 95 characters, but in reality the each tab counted for a single character/byte and so it evaluated to a length of 32 instead. See the following test code:
```
33> {ok, Line} = read_a_line().
{ok,<<"\t\t\t\t\t\t\t\t\t    io:format(\"Hello\").\n">>}
34> byte_size(Line).
33
```

The newline character is still included in the above example which is why it actually evaluates byte length to 33 instead of the more accurate line length of 32.

I will update my pull request and, for the moment, exclud checking tab-prefixed lines since the behavior appears to be "broken" from a user standpoint.